CONTRIBUTING.md

```diff
@@ -1,9 +1,21 @@
 # Contributing
 
-Lumigator is still in early stages of development and as such will have 
-large variance between even PATCH versions in terms of feature development. 
-While we're in this phase, we're accepting PRs for clarification in documentation. 
+Lumigator is still in early stages of development and as such will have
+large variance between even PATCH versions in terms of feature development.
 
-## Project structure
+We do want encourage both suggestions and contributions from the community in the near future.
 
-The lumigator project contains several packages: the `lumigator/python/mzai/backend` package that powers the lumigator server functionality, the `lumigator/python/mzai/schemas` package containing the formal schemas for communication with the server through the REST API, and the `lumigator/python/mzai/sdk` package abstracting the REST API for Python applications. Each package holds its own `pyproject.toml` definition.
\ No newline at end of file
+Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.
+
+As long as this text is here, we are able to accept:
+
+* PRs for **bugfixes** and clarification in **documentation** only;
+* [bug reports](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+), gladly: please include as much information as possible;
+* [suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
+
+
+# Whare to go
+
+* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/), describing all the components and including usage guides;
+* [open issues](https://github.com/mozilla-ai/lumigator/issues);
+* [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md) options.
```

Comment:
You are linking to `CONTRIBUTING.md` but the user is already in this file. I suggest the following:

```suggestion
* [Contribute](https://github.com/mozilla-ai/lumigator/contribute): Explore the list of "good first issues" to start contributing.
```